### DISCUSSION TITLE
Adding a new cable filtering

### DISCUSSION THREAD

**Discussion by mkarel** [netbox-community/netbox#17450] [Ideas]
When adding a new cable, when you chose the type (Interface, Front Port, Power port..) it would be amazing if the device drop-down would filter out devices that don't have the components necessary to make the connection.  Example if I chose to make a connection from an interface to a front port, only provide a list of devices that have a front port in the drop-down.

**Comment by ydc-dg** [comment]
In addition it should per default filter devices in the same rack or at least same location.
In 99% It makes not much sence that I get devices listed wich are installed in a different Site or location.
